pkg/fakedata/generator.go

```diff
@@ -16,66 +14,51 @@ import (
 
 // A Generator is a func that generates random data along with its description
 type Generator struct {
-	Func func(Column) string
+	Func func() string
```

Comment:
This is the main reason why I'm happier with this design that the current master :)